# Request: kosong
---
geometry_msgs/Point robot_pos
geometry_msgs/Point target_pos

